include/stronk/unit.h

```diff
@@ -315,9 +254,9 @@ constexpr auto operator/(const T& a, const T& b) noexcept -> T
 template<identity_unit_like T, unit_like B>
 constexpr auto operator/(const T& a, const B& b) noexcept
 {
-    return NewUnitType<typename B::underlying_type,
-                       UnitTypeLists<typename B::divided_part, typename B::multiplied_part>> {a.template unwrap<T>()
-                                                                                              / b.template unwrap<B>()};
+    using new_dimensions = typename B::dimensions_t::negate_t;
+    using new_unit = typename unit_lookup<new_dimensions, typename B::underlying_type>::type;
+    return new_unit {a.template unwrap<T>() / b.template unwrap<B>()};
 }
```

Comment:
Check if we can skip having the identity_unit_like overloads of these function (also for multiplication)